src/styles/Hero.css

```
@@ -303,23 +305,52 @@ Link{
   }
-  .hero-images{
-    margin-top: 170px;
+  
+  .countdown-number{
+    color: #ed444b;
+ 
   }
+  
+  .countdown p{
+  
+  }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)